src/components/main/Layout.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import MainNavBar from './MainNavBar';
+
+const Layout = ({ employee, children }) => {
+    let statusMenu = "active-btn";
+    let statusKitchen = "nav-btn";
+    let statusOrders = "nav-btn";
+    return (
+        <div className="main-container">
+            <MainNavBar 
+            employee={employee}
+            statusMenu={statusMenu}
+            statusKitchen={statusKitchen}
+            statusOrders={statusOrders} 
+            />
+            {children}
+        </div>
+    )
+}
```

Comment:
```suggestion
const Layout = ({ employee, children }) => {
    return (
        <div className="main-container">
            <MainNavBar 
              employee={employee}
              statusMenu="active-btn"
              statusKitchen="nav-btn"
              statusOrders="nav-btn" 
            />
            {children}
        </div>
    )
}
```

BTW, he notado que estamos usando `function` (declaraciones de funciones) para crear nuestros componentes, en este caso estamos usando arrow functions. Mi recomendación es consistencia, si vamos a optar por uno la idea sería mantenerlo en todo nuestro código al menos que haya alguna característica especial de uno u el otro que deseamos tomar provecho.

Reply:
Es cierto y ha sido un poco a propósito. Lo cierto es que he usado ambos porque de pronto no me acostumbraba a 'la notación'. Pero entiendo lo de la consistencia. Gracias. :)